src/components/topNav/primaryTopNav/PrimaryTopNav.module.css

```diff
@@ -4,6 +4,7 @@
   --conduction-primary-top-nav-background-color: var(--skeleton-color-secondary-3);
   --conduction-primary-top-nav-background-color-hover: var(--skeleton-color-secondary-5);
   --conduction-primary-top-nav-dropdown-border-radius: var(--skeleton-border-radius-md);
```

Comment:
I'd expect the hover color to be the same as the active color. In current implementation, that would result to: `hsla(0,0,100%,.3)`.

The current token you've set (`--skeleton-color-secondary-5`) does not exist within the component context and is therefore not a good default to use (when implemented in a context which **does not** set the value to this key, there will be no hover color).

Above issue is what's happening within the `web-app` repository (Open Catalogi), we're implementing this without setting the value, resulting in nothing happening. We'd expect some default value to take over here.